### ISSUE TITLE
Import EAG Reconnect 2021 data and EA Fellowship

### ISSUE THREAD

**Issue by viktor-yunenko** [rtcharity/eahub.org#1242]
EAG checklist:

* [x] merge two different datasets into one
* [x] check whether the users who we need to import updated their profiles data after the EAG
* [x] parse the linkedin_url field
* [x] create twitter and calendly fields
* [ ] import the images - with `requests`
* [ ] handle images with lower resolutions - disable `upscale`? add a db flag?
* [ ] import 10%
* [ ] setup the new email provider
* [ ] test the newsletter on my user

Mapping: https://docs.google.com/spreadsheets/d/1Y_GKftJeWh5LBYflnPJwrFrLN6BgIiTGzM-CA-fGK8I/edit#gid=189592928

**Comment by viktor-yunenko** [comment]
Hi @sebgrebe, I received the latest update from Barry and it needs a bit of handling. I broke it down into the steps here.

In case you won't be around for a review during the weekdays I'm thinking about force merging those patches in, would that be ok for you?

**Comment by sebgrebe** [comment]
@victor-yunenko, absolutely, no problem. I'm gone from Wednesday, back on Sunday morning
